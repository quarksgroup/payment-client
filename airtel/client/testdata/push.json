{
  "data": {
    "transaction": {
      "id": "QUARKS350273CSD55X",
      "status": "Success."
    }
  },
  "status": {
    "response_code": "DP00800001006",
    "code": "200",
    "success": true,
    "result_code": "ESB000010",
    "message": "Success."
  }
}