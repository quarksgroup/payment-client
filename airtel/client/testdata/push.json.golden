{
  "ref": "QUARKS350273CSD55X",
  "status": true,
  "message": "Success.",
  "result_code": "ESB000010"
}